test/tag-spec/TestTagSpec.gmk

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+#----------------------------------------------------------------------
+#
+# checks that the env vars and system properties given to tests
+# are documented in tag-spec.html, and vice-versa
+
+$(BUILDTESTDIR)/TestTagSpec_env.ok: \
+		$(JTREG_TAGSPEC) \
+		$(JAVAFILES.com.sun.javatest.regtest-tools)
+	$(GREP) -oh '.put("TEST[^"]*' $(JAVAFILES.com.sun.javatest.regtest-tools) | \
+		$(SED) -e 's/.*"//' | $(SORT) -u > $(@:%.ok=%.src)
+	$(GREP) -o '<code>TEST[^<]*</code>' $(JTREG_TAGSPEC) | \
+		$(SED) -e 's/<[^>]*>//g' | $(SORT) -u > $(@:%.ok=%.spec)
+	$(DIFF) $(@:%.ok=%.src) $(@:%.ok=%.spec)
+	echo "test passed at `date`" > $@
+
+$(BUILDTESTDIR)/TestTagSpec_sysProps.ok: \
+		$(JTREG_TAGSPEC) \
+		$(JAVAFILES.com.sun.javatest.regtest-tools)
+	$(GREP) -oh '\(.put(\|put(map, \)"test.[^"]*' $(JAVAFILES.com.sun.javatest.regtest-tools) | \
+		$(GREP) -v "test.class.path.prefix" | \
```

Comment:
Why do we want to ignore this sysprop?  It's not defined in the tag spec, but why is it defined... convenience?